### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by Klay4** [cdmochi/NearbyMessagingPoc#1]
It doesn't find nearby devices...
I've tried to build and install the application on two different android devices, allowed the permissions and checked both the bindings but nothing happens (the bluetooth is on)

**Comment by cdmochi** [comment]
Have you tried adding the nearby API key to this part of the manifest yet?
 ` <meta-data
            android:name="com.google.android.nearby.messages.API_KEY"
            android:value="USER YOUR OWN API KEY" />`

**Comment by Klay4** [comment]
yeah, I've already generated (here https://console.developers.google.com/flows/enableapi?apiid=copresence&keyType=CLIENT_SIDE_ANDROID&reusekey=true) and added the key
